doc/topics/community/index.rst

```diff
@@ -0,0 +1,97 @@
+========================
```

Comment:
Should also probably add something like `.. _salt-community:` to the top here as an internal reference label